doc/releases/changelog-dev.md

```diff
@@ -4,21 +4,25 @@
 
 <h3>New features since last release</h3>
 
+* Added a new function `qml.new_parameters` that creates a copy of an operator with new parameters
```

Comment:
```suggestion
* Added a new function `qml.bind_new_parameters` that creates a copy of an operator with new parameters
```

Reply:
Actually:
```suggestion
* Added a new function `qml.ops.bind_new_parameters` that creates a copy of an operator with new parameters
```

Or will it have to be `qml.ops.functions.bind_new_parameters`?